python-api/products_ranker_api/spiders/aliexpress_httpx.py

```
@@ -0,0 +1,134 @@
+    # Price and currency
+    price_node_container = card.css_first(".jr_kr")
+    price, currency = 0.0, "USD"
+    if price_node_container:
+        spans = price_node_container.css("span")
+        if spans:
+            currency = spans[0].text(strip=True) if spans[0] else "USD"
+            price_text = "".join([s.text(strip=True) for s in spans[1:]])
+            try:
+                price = float(price_text)
+            except ValueError:
+                pass
```

Ruff (0.8.2)

Use `contextlib.suppress(ValueError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(ValueError)`

(SIM105)